src/bindings/pygaia/scripts/classification/README.md

```diff
@@ -0,0 +1,51 @@
+# Classification script
+
+This folder contains scripts for training SVM models. Given a dataset, several parameters are tried through a grid search process using n-fold cross-validation.
+A final model is trained with all the data (without validation set) featuring the best parameters combination in terms of accuracy. 
+
+## Main routines
+`train_model.py` is the main script. It can be both, imported as a module or run as an executable. This is the syntax for the latter case:
+`train_model.py  [--seed SEED] [--cluster_mode] groundtruth_file filelist_file project_file project_dir results_model_file `
+
+Parameters:
+`groundtruth_file` is yaml file containing a relation between keys and labels. Refer to `groundtruth_example.yaml` in this folder for an example.
+`filelist_file` is a yaml file containing a relation between keys and feature file paths. Refer to `filelist_example.yaml` in this folder for an example.
+`project_file` is the path where the project configuration file will be stored.
+`project_dir` is the base path for the files generated during the training process.
+`results_model_file` is the path to the file where the best model ranking the best performance will be stored.
+`--seed` is an optional numeric parameter that controls the data splits for the n-fold cross-validation. If not specified, folds are stochastically chosen.
+`--cluster_mode` is a flag to explicitly use the subprocess module to open a new python process for each subtask.
+
+Alternatively, the script `train_model_from_sigs.py` only requires a path to a folder containing subfolders representing the classes with the features.
+For instance, given the path `base/path/` the script will expect a set of files organized in subfolders as `base/path/class-a/`, `base/path/class-b/`...
+The scripts generates the `groundtruth_file` and the `filelist_file` and calls `train_model.py`.
+`train_model_from_sigs.py [--force] [--seed SEED] [--cluster_mode] input_folder output_folder project_name `
+
+Parameters:
+`input_folder` the base path to the subfolders with features.
+`output_folder` the project folder where to store all the generated files.
+`--seed` is an optional numeric parameter that controls the data splits for the n-fold cross-validation. If not set, folds are stochastically chosen.
+`--cluster_mode` is a flag to explicitly use the subprocess module to open a new python process for each subtask.
+`--force` Whether to retrain a model if output_folder already has one.
+
+
+Parameters:
```

Comment:
what's the difference between this section and the one just above it?

Reply:
This should be consolidated.